code/modules/mob/living/carbon/human/life/handle_disabilities.dm

```diff
@@ -73,7 +73,17 @@
 						"FUCK IT; KISSYOUR ASSES GOOD BYE DEAD MEN! I AM SELFDESTRUCKTING THE STATION!!!!", \
 						"OMG I SED LAW 2 U FAG MOMIM LAW 2!!!"))
 				if(3)
-					emote("drool")
+					// We need to find a mob in range for that ; could be useful for custom emotes that take one random mob at random as an argument, maybe ?
+					var/list/L
+					var/retardation
+					for(var/mob/M in range(1, src))
+    					L.Add(M)
+    				var/mob/mob1 = L[1]
+    				if(!list.len)
+    					retardation = "pecks \the [mob1]."
```

Comment:
There should be a way that you have the retarded person bite the mob in question if they're that close. Might be worth a try to develop the idea a little

Reply:
I find a bit heavy for just a brain damage proc to call not only a list sorting and another proc such as bite that might be more complex.
I'll see to it. For now I just want to correct the ridiculous code I posted